src/clr-icons/shapes/technology-shapes.ts

```diff
@@ -1112,7 +1118,10 @@ export const TechnologyShapes: any = {
     "hard-drive": ClrShapeHardDrive,
     "hard-drive-disks": ClrShapeHardDriveDisks,
     "nvme": ClrShapeNVMe,
-    "ssd": ClrShapeSSD
+    "ssd": ClrShapeSSD,
+    "bluetooth": ClrShapeBluetooth,
+    "no-bluetooth": ClrShapeNoBluetooth,
```

Comment:
isn't `bluetooth-off` better here? You do the same with `alarm-off`. Two different naming conventions. We need to choose one

Reply:
I like Aditya's suggestions. 👍

Reply:
😎